src/OpenFga.Sdk.Test/Api/OpenFgaApiTests.cs

```diff
@@ -341,6 +354,7 @@ public class OpenFgaApiTests : IDisposable {
             var openFgaApi = new OpenFgaApi(config, httpClient);
 
             var response = await openFgaApi.ReadAuthorizationModels(null, null);
+            var response2 = await openFgaApi.ReadAuthorizationModels(null, null);
```

Comment:
1. Do we have any test testing token expire?
2. Do we want to break the assertion into multiple steps (i.e., for the first response, there should be a) Get Token b) GetAuthModel.  Then for second response, there should be only GetAuthModel and do not expect getToken).  If this is not possible, at least note what we are expecting and why.

Reply:
Added the test for (2)

Reply:
For (2) we have other tests covering this